zaza/openstack/charm_tests/test_utils.py

```diff
@@ -133,6 +133,29 @@ def setUpClass(cls, application_name=None, model_alias=None):
             model_name=cls.model_name)
         logging.debug('Leader unit is {}'.format(cls.lead_unit))
 
+    def config_current(self, application_name=None, keys=None):
```

Comment:
new method here to get the current config from an application.  You can supply `keys` if you only want certain items from the current config.